Automattic/wp-calypso
# SSO: Add invite user button option

Related to #

## Proposed Changes

* We want to align as much as possible wp-admin and calypso feature, this PR makes it possible to invite not connected users to join your site by adding an invite button to the user list

<img width="726" alt="image" src="https://github.com/Automattic/wp-calypso/assets/2653810/1500a8b9-6ab1-4faa-aac5-357ebe0b75dc">


## Testing Instructions



* Pull and run this branch or use live link
* Navigate to {LIVE_LINK}/people/team/{YOUR_SITE}
* There you should be able to invite not connected users to join your site
* You should also be allowed to revoke invited users from the pending list users
* You should not see the same users in both lists
* Invite users from wp-admin and check if both lists are persistent, you should see them as pending in Calypso

## Pre-merge Checklist
